src/filesystem.hpp

```diff
@@ -121,14 +121,13 @@ static const blacklist_pattern_list default_blacklist{
 };
 
 /**
- * Populates 'files' with all the files and
- * 'dirs' with all the directories in dir.
- * If files or dirs are nullptr they will not be used.
- *
- * mode: determines whether the entire path or just the filename is retrieved.
- * filter: determines if we skip images and sounds directories
- * reorder: triggers the special handling of _main.cfg and _final.cfg
- * checksum: can be used to store checksum info
+ * @param dir The directory to examine.
```

Comment:
I still think this doc comment should include a brief summary.

Reply:
So just something like `Loads a list of files and-or directories from @a dir`?

Reply:
Never mind, I see you had a general description on your previous comment.

Reply:
Either mine or yours sounds fine.